src/frameworks.js

```diff
@@ -19,18 +19,21 @@
 'use strict';
 
 var useRequireEnsure = false;
-if (typeof module !== 'undefined' && module.require) {
-  // node.js - disable worker and set require.ensure.
-  PDFJS.disableWorker = true;
-  if (typeof require.ensure === 'undefined') {
-    require.ensure = require('node-ensure');
+if (typeof module !== 'undefined' && module.exports !== 'undefined') {
```

Comment:
Mistake here: `module.exports !== 'undefined'` is always true, so it can be removed.